### DISCUSSION TITLE
Problem with external service: pyzor

### DISCUSSION THREAD

**Discussion by ttrueten** [rspamd/rspamd#3828] [Q&A]
Hello everybody,
For various reasons, I would like to install the pyzor service in rspamd. I have installed pyzor 1.0.0 on our ubuntu server 20.04 via the repositories and installed it as an external service in rspamd [according to the specifications](https://rspamd.com/doc/modules/external_services.html#pyzor-specific-details).

```
pyzor {
  # default pyzor settings
  servers = "127.0.0.1:5953"
}
```

Systemd socket and service are enabled as described, pyzor itself works, I can scan through mails.

However, rspamd produces the following error message when it is started:

```
Jul 27 09:34:35 mailsystem rspamd[12772]: <yqzuhc>; lua; external_services.lua:124: unknown external scanner type: pyzor
Jul 27 09:34:35 mailsystem rspamd[12772]: <yqzuhc>; lua; external_services.lua:206: cannot add rule: "pyzor"
```

Other services like razor / DCC work fine, what can I do?

TIA,
Thomas

**Comment by fatalbanana** [comment]
This was very recently added; stable build is too old & it looks like so is the current snapshot.
